locale/en-us/run_mall_parser.intent

```diff
@@ -0,0 +1,13 @@
+neon start mall parser
```

Comment:
You might have an easier time parsing store names by refactoring this intent to parse the store for you, ie:
```
where (is|can i find) {{shop}}
where is {{shop}}(| located)
```
Example from the translation skill: https://github.com/NeonGeckoCom/skill-translation/blob/master/locale/en-us/vocab/translate_phrase.intent